user/enterprise/ssl-certificate-management.md

```diff
@@ -88,131 +88,167 @@ Upon restart you can then verify whether SSL verification is now working. If you
 
 ## Using a Let's Encrypt SSL Certificate
 
-Travis CI Enterprise works well together with a Let's Encrypt SSL Certificate. To obtain one for your domain, please follow the steps below.
+You can use a certificate from [Let's Encrypt](https://letsencrypt.org/) instead of a self-signed certificate or a certificate purchased from a trusted certificate authority. Certificates from Let's Encrypt are free and behave the same as those purchased from a trusted certificate authority. More information about Let's Encrypt can be found [here](https://letsencrypt.org/about/).
```

Comment:
```suggestion
You can use a certificate from [Let's Encrypt](https://letsencrypt.org/) instead of a self-signed certificate or a certificate purchased from a trusted certificate authority. Certificates from Let's Encrypt are free and behave the same as those purchased from a trusted certificate authority.
```

It is best to avoid `[here]` for link text, terrible for screen readers and SEO. As we've already linked let's Encrypt, we can just delete.

Reply:
Got it, thanks for the explanation. I'll remove it. 👍

Reply:
Addressed in 192be263